go/grpcweb/websocket_wrapper.go

```diff
@@ -24,40 +24,34 @@ type webSocketResponseWriter struct {
 	flushedHeaders  http.Header
 	timeOutInterval time.Duration
 	timer           *timer.Timer
+	context         context.Context
 }
 
-func newWebSocketResponseWriter(wsConn *websocket.Conn) *webSocketResponseWriter {
+func newWebSocketResponseWriter(wsConn *websocket.Conn, ctx context.Context) *webSocketResponseWriter {
```

Comment:
Context is by convention the first parameter to a function
```suggestion
func newWebSocketResponseWriter(ctx context.Context, wsConn *websocket.Conn) *webSocketResponseWriter {
```